lib/imapSimple.js

```diff
@@ -92,6 +92,40 @@ ImapSimple.prototype.openBox = function (boxName, callback) {
     });
 };
 
+/**
+ * Close a mailbox
+ *
+ * @param {boolean} autoExpunge If autoExpunge is true, any messages marked as Deleted in the currently open mailbox will be remove
+ * @param {function} [callback] Optional callback, receiving signature (err)
+ * @returns {undefined|Promise} Returns a promise when no callback is specified, resolving to `boxName`
+ * @memberof ImapSimple
+ */
+ImapSimple.prototype.closeBox = function (autoExpunge, callback) {
```

Comment:
to fully make this optional, you can set a default value in the function signature

`function (autoExpunge = true, callback) {`

This is [supported in Node 6+](https://node.green/#ES2015-syntax-default-function-parameters) so I think it's legit

Reply:
Then again, if you were able to avoid using the default function param and handle this some other way, we could release this as a minor version bump. I'm good either way.